mautic/mautic
# Clean fields during the get leads from integration

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.0 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
I've notice Mautic clean email address If contains specific characters like ç,č,š,ď....  But during the fetch leads from integration this create a lot of duplicates contacts. Mautic clean email address during the create, then was not able to find already created contact. This PR fixed it.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Setup hubspot integration
3. Create contact with email françois@something.com in Hubspot
4. Try sync 3 times `php bin/console mautic:integration:fetchleads --integration=Hubspot`
5. You should see 3 created contact `franois@something.com` (without ç in email address)
6. After patch you should sync franois@something.com without create integration 




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9366"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
